src/Nri/Ui/MediaQuery/V1.elm

```diff
@@ -73,3 +78,22 @@ quizEngineMobile =
 quizEngineBreakpoint : Css.Px
 quizEngineBreakpoint =
     px 750
+
+
+{-| Styles using the `narrowMobileBreakPoint` value as the maxWidth
+-}
+narrowMobile : MediaQuery
+narrowMobile =
+    only screen
+        [ --`minWidth (px 1)` is for a bug in IE which causes the media query to initially trigger regardless of window size
```

Comment:
We might be able to get rid of IE fixes, since we no longer support it, but not sure if that violates "first, do no harm"

Reply:
I don't know if violates. I am ok with it, the only thing is that it is a breaking change in the API behavior, and breaking changes are kind of evil 😈 . cc\ @tesk9 what are your thoughts on that?

Reply:
Eh I think either choice is fine. We'll almost certainly remove it in `MediaQuery.V2`. 🤷‍♀️ It's easiest not to change I guess, but we don't _need_ it